Assets/SEE/UI/Drawable/DrawableFileBrowser.cs

```diff
@@ -35,160 +32,99 @@ public class DrawableFileBrowser : MonoBehaviour
             }
 
-            /// Ensures that only configuration files can be selected.
-            FileBrowser.SetFilters(false, Filenames.ConfigExtension);
-
-            /// Opens the file browser.
-            yield return FileBrowser.WaitForSaveDialog(FileBrowser.PickMode.Files, false, initPath, null, title, "Save");
-
-            /// Enables the short cuts again.
-            SEEInput.KeyboardShortcutsEnabled = true;
+            PathPicker.GetPath(title, false, initPath, HandleFileBrowserSuccess, () => { }, Filenames.DrawableConfigExtension);
+        }
 
-            if (FileBrowser.Success)
+        /// <summary>
+        /// Called when the file browser has successfully chosen a file.
+        /// Sets the path to the chosen file and refreshes the UI canvas.
+        /// </summary>
+        /// <param name="path">the path the user selected</param>
+        private void HandleFileBrowserSuccess(string path)
+        {
+            if (!string.IsNullOrWhiteSpace(path))
             {
-                /// Sets the file path upon success.
-                SetPath(FileBrowser.Result[0]);
+                SetPath(path);
             }
 
             /// Refreshes the UI canvas to prevent display issues.
             UICanvas.Refresh();
         }
 
         /// <summary>
-        /// Method to load an image
+        /// Method to load an image. The file is selected by the user by way
+        /// of the <see cref="FileBrowser"/>.
         /// </summary>
         public void LoadImage()
-        {
-            SEEInput.KeyboardShortcutsEnabled = false;
-            StartCoroutine(ShowLoadImageDialogCoroutine());
-        }
-
-        /// <summary>
-        /// Coroutine that enables the file browser to chose a file.
-        /// </summary>
-        /// <returns>nothing</returns>
-        private IEnumerator ShowLoadImageDialogCoroutine()
         {
             DrawableConfigManager.EnsureDrawableDirectoryExists(ValueHolder.ImagePath);
             initPath = ValueHolder.ImagePath;
-            string title = "Load an image";
 
             /// Ensures that only PNG or JPG files can be selected.
-            FileBrowser.SetFilters(true, Filenames.PNGExtension, Filenames.JPGExtension);
-
-            /// Opens the file browser.
-            yield return FileBrowser.WaitForLoadDialog(FileBrowser.PickMode.Files, false, initPath, null, title, "Load");
-
-            /// Enables the short cuts again.
-            SEEInput.KeyboardShortcutsEnabled = true;
-
-            if (FileBrowser.Success)
-            {
-                /// Sets the file path upon success.
-                SetPath(FileBrowser.Result[0]);
-            }
-
-            /// Refreshes the UI canvas to prevent display issues.
-            UICanvas.Refresh();
+            PathPicker.GetPath("Load an image", false, initPath, HandleFileBrowserSuccess, () => { },
+                                new string[] { Filenames.PNGExtension, Filenames.JPGExtension });
```

Comment:
Note that JPEG files can have either `.jpg` or `.jpeg` as their file extension, right now we are only catching `.jpg` files.

Reply:
I added these.